compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -370,10 +370,18 @@ fn update_target_reliable_float_cfg(sess: &Session, cfg: &mut TargetConfig) {
     let target_env = sess.target.options.env.as_ref();
     let target_abi = sess.target.options.abi.as_ref();
     let target_pointer_width = sess.target.pointer_width;
+    let version = get_version();
 
     cfg.has_reliable_f16 = match (target_arch, target_os) {
         // Selection failure <https://github.com/llvm/llvm-project/issues/50374>
         ("s390x", _) => false,
+        // LLVM crash without neon <https://github.com/llvm/llvm-project/issues/129394> (now fixed)
+        ("aarch64", _)
+            if !sess.target_features.iter().any(|f| f.as_str() == "neon")
```

Comment:
Maybe this? But I'm not sure of the difference.

```suggestion
            if !cfg.target_features.iter().any(|f| f.as_str() == "neon")
```

Reply:
(also if we do stick to `sess`, that's a set, so `contains` may be better.)

Reply:
Good thought, fingers crossed that it is that easy